packages/config/src/ConfigService/MutableConfigService.ts

```diff
@@ -0,0 +1,56 @@
+import {
+  AnyChain,
+  Asset,
+  EvmParachain,
+  EvmParachainConstructorParams,
+  Parachain,
+  ParachainConstructorParams,
+} from '@moonbeam-network/xcm-types';
+import { AssetConfig } from '../types/AssetConfig';
+import { ChainConfig } from '../types/ChainConfig';
+import { ConfigService } from './ConfigService';
+
+export class MutableConfigService extends ConfigService {
```

Comment:
I really like that you created `ConfigService`. I'm just not sure about `MutableConfigService`. Maybe I would just add `updateAssets` (which I would rename to `updateAsset`), `updateChains` (`updateChain`), `updateChainConfig` to `ConfigService`. Other methods I wouldn't add.
If users need to change a few `chains`, `assets`, `chainConfigs` that should be enough  and if they need to more they can always create a new instance of `ConfigService`. Or you need them?

Reply:
@ekenigs We can drop MutableConfigService :) .. You're right, user can always inherit from ConfigService & write 'extra' utility functions on top of that. I don't really use them, frankly i ended u with following:

```
function configure(configService: MutableConfigService) {
  assets.forEach((asset) => {
    configService.updateAssets(asset);
  });

  chains.forEach((chain) => {
    configService.updateChains(chain);
  });

  chainsConfig.forEach((config) => {
    configService.updateChainConfig(config);
  });
}
```
It's was more like nice to have for new folks to ease their lives :) I'm just thinking maybe one more function `reset()` to wipe the entire config if you want to setup the configuration from scratch .. Reset can be facade calling map.clear() for both chains, assets, chainConfigs. WDYT ?

Reply:
;) Alright .. Should be done. Changeset updated.